actions/glossary-search.php

```diff
@@ -0,0 +1,169 @@
+      margin-left: -15px;
+      margin-right: -15px;
+   }
+
+   .key .keys {
+      background-color: #e6e6e6;
+      padding-left: 10px;
+      padding-right: 10px;
+      padding-top: 8px;
+      padding-bottom: 8px;
+      cursor: pointer;
+      height: 35px;
+      width: 35px;
+   }
+
+   .keys .key>a,
+   .globalsite.cmp-globalsite-glossarysearch .keys .numkey>a {
+      font-size: 17px;
+      font-weight: 500;
+      line-height: 20px;
+      text-align: center;
+      color: #0052d6;
+      display: block;
+      cursor: pointer;
+   }
+
+   .key {
+      display: flex;
+      flex-wrap: wrap;
+   }
+
+   .letter {
+      width: 33.33%;
+
+   }
+
+   .column {
+      list-style: none;
+   }
+
+   .a {
+      font-weight: bold;
+   }
+</style>
+<!-- <li><span aria-current="page" class="page-numbers current">3</span></li> -->
+
+<article id="glossary-search">
+  <div class="emt-container-inner">
+    <div class="root responsivegrid">
+      <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+        <div class="responsivegrid aem-GridColumn aem-GridColumn--default--12">
+          <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+            <div class="globalsite cmp-globalsite-hero js-hero-check oldviz aem-GridColumn aem-GridColumn--default--12">
+              
+               <section>
+                  <!-- // page header -->
+                  <div class='page-header'>
+                     <h1 data-en-heading="Glossary" style="color:#002856">Vula Glossary</h1>
+                  </div>
+               </div>
+            </div>
+            <label for="word">Languages list</label> &nbsp; &nbsp; &nbsp; &nbsp;
+            <select name="languages" id="languageSelect">
+               <?php
+               foreach ($languages as $arr => $value): ?>
+                  <option>
+                     <?= $value['language']; ?>
+                  </option>
+               <?php endforeach; ?>
+            </select> <br>
+            <div id="termsContainer">
+               <!-- Display terms here -->
+            </div>
+            <input type="text" id="searchBox" name="searchBox" placeholder="Search Vula Glossary" value="">
+            <button type="submit" class="btn btn-primary" id="searchButton">
+               Search
+            </button>
+            <div id="searchResults"> </div>
+         </form>
+         <div class="row">
+            <div class="found-result col-xs-12 p-small" style="display:none"></div>
+         </div>
+         <div class="row keys hidden-sm hidden-xs">
+            <div>
+               <?php
+               echo '<button>0-9</button>';
+               // Array of alphabets
+               $alphabets = range('A', 'Z');
+               foreach ($alphabets as $alphabet) {
+                  echo '<button type="button" data-term_id="{{searchTerm}}"  >' . $alphabet . '</button>';
+               }
+               ?>
+            </div>
+            &nbsp; <div class="col-xs-12">
```

Comment:
remove this character '&nbsp;' - if you need spacing then rather use margin or padding